plugin.video.rocketbeans/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.rocketbeans" version="1.0.0" name="Rocket Beans TV" provider-name="Razzeee">
+    <requires>
+        <import addon="xbmc.python" version="2.14.0" />
+        <import addon="plugin.video.youtube" version="5.2.1" />
+        <import addon="script.module.routing" version="0.2.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="de_de">Rocket Beans TV | 24/7 Entertainment</summary>
```

Comment:
I think you are required to provide the same tags in en_GB as well (for the website and addon browser)